src/controllers/capacity/index.ts

```diff
@@ -138,6 +138,19 @@ export class CapacityController extends Controller {
     if (!creation) throw new NotFound('creation')
     return creation
   }
+
+  @Response<NotFound>(404, 'Items not found.')
+  @SuccessResponse('200')
+  @Get('{capacityId}/creation/')
+  public async getTransactionsFromCapacity(@Path() capacityId: UUID): Promise<TransactionResponse[]> {
+    const [capacity] = await this.db.getDemand(capacityId)
+    if (!capacity) throw new NotFound('capacity')
+
+    const creations = await this.db.getTransactionsFromCapacityID(capacityId)
+    if (!creations) throw new NotFound('capacity')
```

Comment:
can remove checking if `creations` is falsy, there's no chance of `404`ing at this point because you've already checked the capacity ID. I think the only way `creations` would be a false value like `undefined` would be if there's some random unexpected error in which case the error handler will catch it and response with a `500`